tests/System/integration/administrator/components/com_privacy/Consent.cy.js

```diff
@@ -1,8 +1,12 @@
 describe('Test in backend that the privacy consent component', () => {
   beforeEach(() => cy.doAdministratorLogin());
   afterEach(() => {
-    cy.task('queryDB', 'DELETE FROM #__privacy_consents');
-    cy.task('queryDB', "DELETE FROM #__users WHERE name = 'test user'");
+    cy.task('queryDB', 'TRUNCATE #__privacy_consents');
+    cy.task('queryDB', 'DELETE FROM #__users WHERE name LIKE "%test user%"')
```

Comment:
```suggestion
    cy.task('queryDB', "DELETE FROM #__users WHERE name LIKE '%test user%'")
```
The string for the username has to be quoted with single quotes so we have to use double quotes for the complete statement, not vice versa.